build.gradle

```diff
@@ -33,6 +35,17 @@ googleJavaFormat {
   exclude 'build/**'
 }
 
+if (project.hasProperty('ossrhUsername') && project.hasProperty('ossrhPassword') &&
```

Comment:
What if this statement is `false`? Shoud the script fail with en error message like "please configure our ossrhUsername creds in settings.xml"?